src/main/java/reposense/git/GitClone.java

```diff
@@ -84,8 +91,15 @@ public static void clone(RepoConfiguration config) throws GitCloneException {
      */
     public static void cloneBare(RepoConfiguration config, String outputFolderName) throws IOException {
         Path rootPath = Paths.get(FileUtil.REPOS_ADDRESS, config.getRepoFolderName());
-        FileUtil.deleteDirectory(Paths.get(rootPath.toString(), outputFolderName).toString());
-        Files.createDirectories(rootPath);
+        Path outputFolderPath = Paths.get(rootPath.toString(), outputFolderName);
+
+        if (!SystemUtil.isTestEnvironment()) {
+            FileUtil.deleteDirectory(outputFolderPath.toString());
+            Files.createDirectories(rootPath);
```

Comment:
Is it possible that you can add a test to cover this branch here?

Reply:
I don't think so, as this branch can only run if it is a non-test environment. The test environment is defined in an environment variable and [it won't be easy to modify it just for one test](https://stackoverflow.com/questions/580085/is-it-possible-to-set-an-environment-variable-at-runtime-from-java).